spec/features/shelters/user_can_edit_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "When a user clicks update shelter", type: :feature do
+    it "can update shelter details" do
+        shelter_1 = Shelter.create( name:    "4 Paws Rescue",
+                                    address: "6567 W Long Dr.",
+                                    city:    "Littleton",
+                                    state:   "CO",
+                                    zip:     "80123")
+
+        visit "/shelters/#{shelter_1.id}"
+        click_link "Update Shelter"
+        
+        expect(current_path).to eq("/shelters/#{shelter_1.id}/edit")
+
+        fill_in :name, with: "4 Paws Animal Rescue"
+        click_button "Update"
+
+        expect(current_path).to eq("/shelters/#{shelter_1.id}")
+        expect(page).to have_content("4 Paws Animal Rescue")
```

Comment:
Makes sure that after updating it redirects back to the shelters show page. and makes sure that the field has been updated.